src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -69,4 +105,15 @@ public String getExampleInput() {
         return "Albert Einstein and Alan Turing";
     }
 
+    private static boolean contains(final String[] array, final String[] searchTerms) {
```

Comment:
Do not use arrays. Use Collections. [Collections.contains](https://docs.oracle.com/javase/7/docs/api/java/util/Collection.html#contains(java.lang.Object)) is your friend.